kolla/node_custom_config/zun.conf

```diff
@@ -7,15 +7,34 @@ minimum_disk = 0
 default_cpu = 0.0
 default_memory = 0
 default_disk = 0
+{% if enable_zun_compute_k8s | bool %}
+# We support containers, but not capsules when k8s driver is in use.
+container_driver = k8s
+capsule_driver = fake
```

Comment:
This needs to be set as "fake" until the k8s driver advertises support for capsules, even if a no-op.